From 1f4c85d35e39b8347b804c47d021a65afc3caca4 Mon Sep 17 00:00:00 2001
From: wschmidt <wschmidt@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Fri, 27 Jun 2014 16:56:13 +0000
Subject: [PATCH 073/134] 2014-06-27  Bill Schmidt 
 <wschmidt@linux.vnet.ibm.com>

	* gfortran.dg/nint_2.f90: Don't XFAIL for powerpc64le-*-linux*.



git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/branches/gcc-4_8-branch@212077 138bc75d-0d04-0410-961f-82ee72b054a4
---
 gcc/testsuite/ChangeLog              | 4 ++++
 gcc/testsuite/gfortran.dg/nint_2.f90 | 3 ++-
 2 files changed, 6 insertions(+), 1 deletion(-)

diff --git a/gcc/testsuite/ChangeLog b/gcc/testsuite/ChangeLog
index 1932353..3a34783 100644
--- a/gcc/testsuite/ChangeLog
+++ b/gcc/testsuite/ChangeLog
@@ -1,3 +1,7 @@
+2014-06-27  Bill Schmidt  <wschmidt@linux.vnet.ibm.com>
+
+	* gfortran.dg/nint_2.f90: Don't XFAIL for powerpc64le-*-linux*.
+
 2014-06-27  Uros Bizjak  <ubizjak@gmail.com>
 
 	Backport from mainline
diff --git a/gcc/testsuite/gfortran.dg/nint_2.f90 b/gcc/testsuite/gfortran.dg/nint_2.f90
index 7520727..c617a70 100644
--- a/gcc/testsuite/gfortran.dg/nint_2.f90
+++ b/gcc/testsuite/gfortran.dg/nint_2.f90
@@ -4,7 +4,8 @@
 ! http://gcc.gnu.org/ml/fortran/2005-04/msg00139.html
 !
 ! { dg-do run }
-! { dg-xfail-run-if "PR 33271, math library bug" { powerpc-ibm-aix powerpc*-*-linux* *-*-mingw* } { "-O0" } { "" } }
+! { dg-xfail-run-if "PR 33271, math library bug" { powerpc-ibm-aix powerpc-*-linux* powerpc64-*-linux* *-*-mingw* } { "-O0" } { "" } }
+! Note that this doesn't fail on powerpc64le-*-linux*.
   real(kind=8) :: a
   integer(kind=8) :: i1, i2
   real :: b
-- 
1.8.5.5

